site/content/usage/06-vpc-networking.md

```diff
@@ -235,3 +235,14 @@ won't change, and you will still have the option to disable the public endpoint
 the internet. (Source: https://github.com/aws/containers-roadmap/issues/108#issuecomment-552766489)
 
 Implementation notes: https://github.com/aws/containers-roadmap/issues/108#issuecomment-552698875
+
+**Note for using port forwarding with eksctl commands:**
+
+If you access your private cluster locally using port forwarding you'll need to
+override the cluster endpoint that's generated when running eksctl commands.
+
+This can be achieved by setting the following environment variable `KUBECONFIG_CLUSTER_ENDPOINT` when running eksctl commands:
+```
+export KUBECONFIG_CLUSTER_ENDPOINT=https://$KUBERNETES_API_ENDPOINT:8001
```

Comment:
Should this be something like:
```
export KUBECONFIG_CLUSTER_ENDPOINT=https://localhost:8001
```
Does that make sense? (If I understood correctly you are using a local ssh port forwarding right?) Or what other url makes sense?

Reply:
It could be anything really, for example in our case we add our cluster endpoint into `/etc/hosts` as so:
```
127.0.0.1 ABC123XYZ.yl4.ap-southeast-2.eks.amazonaws.com
```

Reply:
aha, is `KUBERNETES_API_ENDPOINT` a variable picked up by kubectl or something? I haven't used that before.

It looks good :) I think we just need a rebase and I can merge it! Thank you! :sparkles:

Reply:
Nah, it's just an environment variable I define to simplify subsequent commands. I've changed the documentation now so instead it just has a placeholder rather than the actual environment variable: https://github.com/weaveworks/eksctl/pull/1746/files#diff-e76866b34ff170d31b648f6bb3918fbeR246

Sweet, I've rebased it now :)